L/libcgroup/build_tarballs.jl

```diff
@@ -0,0 +1,56 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder
+
+name = "libcgroup"
+version = v"3.0"
+
+# Collection of sources required to build libcgroup
+sources = [
+    ArchiveSource("https://github.com/libcgroup/libcgroup/releases/download/v$(version.major).$(version.minor)/libcgroup-$(version).tar.gz",
+                  "8d284d896fca1c981b55850e92acd3ad9648a69227c028dda7ae3402af878edd"),
+    DirectorySource("./bundled"),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd libcgroup-*
+install_license COPYING
+
+atomic_patch -p1 ${WORKSPACE}/srcdir/patches/0001-replace_ac_malloc_realloc.patch
+autoreconf -i
+
+./configure --prefix=${prefix} --build=${MACHTYPE} --host=${target} --disable-pam
+make -j${nproc}
+make install
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = supported_platforms(; experimental=true)
```

Comment:
`experimental` is no-op nowadays :slightly_smiling_face: 
```suggestion
platforms = supported_platforms()
```

Reply:
TIL! Do we still need the Julia 1.6 compat then?

Reply:
Julia compat still defaults to 1.0, although we can consider switching it to 1.0 at some point.

Reply:
On the other hand being explicit makes it easier to catch things like #5842 :slightly_smiling_face: